src/components/Mutations/FineDetailModal.vue

```diff
@@ -1,28 +1,46 @@
 <template>
   <div class="flex flex-column">
-    <div class="flex flex-row justify-content-between">
-      <p>{{ t('fine.total') }}</p>
-      <p>{{ formatPrice(amount) }}</p>
-    </div>
-    <div class="flex flex-row justify-content-between">
-      <p>{{ t('fine.from') }}</p>
-      <p>{{ `${firstName} ${lastName}` }}</p>
-    </div>
-    <div class="flex flex-row justify-content-between">
-      <p>{{ t('fine.type') }}</p>
-      <p>{{ t('fine.fine')  }}</p>
-    </div>
-    <div class="flex flex-row justify-content-between">
-      <p>{{ t('fine.description') }}</p>
-      <p>{{ description }}</p>
-    </div>
+    <span>
+    {{ new Date($props.fine.createdAt!!).toLocaleString('nl-NL', {
+      dateStyle: 'short',
+      timeStyle: 'short'
+    }) }}
+    </span>
+      <span>{{ $t("transactions.fineDescr") }}</span>
+      <br>
+      <DataTable
+        :value="[{
```

Comment:
Hier is uit m'n hoofd een shorthand voor.

Reply:
voor welke precies? De <br>?